src/control/system/raft/mocks.go

```diff
@@ -1,5 +1,5 @@
 //
-// (C) Copyright 2020-2022 Intel Corporation.
+// (C) Copyright 2020-2023 Intel Corporation.
```

Comment:
Looks like copyright updated without any other changes

Reply:
Oops, I had made a change to that file, committed it, then backed out the changes, and forgot to revert the copyright change.